packages/react-scripts/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # `backpack-react-scripts` Change Log
 
+## 9.0.4
```

Comment:
As you are adding a library here could you update the version to be 9.1.0? As this will be a minor change

Reply:
> As you are adding a library here could you update the version to be 9.1.0? As this will be a minor change

Yes, `v9.1.0` is more suitable for that.